tests/components/admin/simulation/simulation-page.test.tsx

```
@@ -0,0 +1,64 @@
+		expect(screen.getByText("No simulations yet")).toBeTruthy();
+		expect(
+			screen.getByText("Start a new simulation by pasting a transcript"),
+		).toBeTruthy();
+	});
+
+	it("opens dialog when 'New Simulation' button is clicked", () => {
+		render(<SimulationPage />);
+
+		expect(screen.queryByTestId("dialog")).toBeNull();
+
+		fireEvent.click(screen.getAllByText("New Simulation")[0]!);
```

Biome (2.4.4)

[error] Forbidden non-null assertion.

(lint/style/noNonNullAssertion)